CHANGELOG.md

```diff
@@ -1,715 +1,819 @@
-
-### Acknowledgements
-
-Thanks to [smortex](https://github.com/smortex) and [Heidistein](https://github.com/Heidistein) who have contributed to this release 🎉!
-
-## [4.9.0] - 2021-09-09
-
-### Summary
-Add Debian 11 support. Add Rocky Linux support for the `install` task. Allow `present` and `latest` as `package_version`. Other fixes and improvements.
-
-### Features
-
-- ([MODULES-11085](https://tickets.puppetlabs.com/browse/MODULES-11085)) Add Fedora 34 support to module ([#564](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/564))
-- Update GPG-KEY-puppet ([#579](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/579))
-- Allow stdlib 8.0.0 ([#576](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/576))
-- ([MODULES-11060](https://tickets.puppetlabs.com/browse/MODULES-11060)) Add Debian 11 to puppet_agent module ([#575](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/575))
-- ([MODULES-11148](https://tickets.puppetlabs.com/browse/MODULES-11148)) Document Windows long paths support ([#573](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/573))
-- Update readme for clarification on Windows agent updates ([#502](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/502))
-- ([MODULES-11113](https://tickets.puppetlabs.com/browse/MODULES-11113)) Allow present and latest as package version ([#565](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/565))
-- ([MODULES-11135](https://tickets.puppetlabs.com/browse/MODULES-11135)) Add Rocky as EL platform in task install_shell.sh ([#571](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/571))
-- ([MODULES-11078](https://tickets.puppetlabs.com/browse/MODULES-11078)) Bump Bolt to 3.x ([#566](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/566))
-- ([MODULES-11112](https://tickets.puppetlabs.com/browse/MODULES-11112)) Add parameter puppet_agent::proxy ([#567](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/567))
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-puppet_agent/compare/v4.14.0...v4.15.0)
 
+### Added
 
-### Bug fixes
+- (PA-5337) PE integration [#654](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/654) ([mhashizume](https://github.com/mhashizume))
 
-- ([MODULES-11123](https://tickets.puppetlabs.com/browse/MODULES-11123)) Avoid loading puppet facts in `install/windows.pp` ([#577](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/577))
-- ([MODULES-11077](https://tickets.puppetlabs.com/browse/MODULES-11077)) Allow all settings to be managed ([#569](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/569))
+### Other
 
-### Acknowledgements
+- (PA-5721) Add puppet8 and drop puppet5 support from AIX [#666](https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/666) ([joshcooper](https://github.com/joshcooper))
```

Comment:
Shouldn't this change make this a major release?

Reply:
We haven't historically bumped the major when dropping EOL versions, e.g. https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/549 Similar to how we drop EOL puppet-agent packages in the 7.x series.